server/db/driver/pg/markets.go

```diff
@@ -83,3 +85,10 @@ func createMarketTables(db *sql.DB, marketUID string) error {
 
 	return nil
 }
+
+// marketSchema replaces the special token symbol character '.' with the allowed
+// PostgreSQL character '$'.
+func marketSchema(marketName string) string {
+	// '$' separator might only work with PostgreSQL.
```

Comment:
https://www.postgresql.org/docs/14/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS

> ...characters in an identifier or key word can be letters, underscores, digits (0-9), or dollar signs ($). Note that dollar signs are not allowed in identifiers according to the letter of the SQL standard, so their use might render applications less portable. The SQL standard will not define a key word that contains digits or starts or ends with an underscore, so identifiers of this form are safe against possible conflict with future extensions of the standard.

Any downsize to using `_` instead?

Reply:
There's already an underscore in the market name, so it would take `dextt.eth_btc` to `dextt_eth_btc` which is ambiguous.